nvflare/lighter/impl/cert.py

```diff
@@ -26,6 +26,8 @@
 from nvflare.lighter.spec import Builder
 from nvflare.lighter.utils import Identity, generate_cert, generate_keys, serialize_cert, serialize_pri_key
 
+MAX_CN_LENGTH = 63
```

Comment:
based on https://github.com/pyca/cryptography/blob/main/src/cryptography/x509/name.py#L66 the max is 64, but this is minor